CHANGELOG.md

```diff
@@ -124,6 +124,7 @@
 * [BUGFIX] Ruler: fix native histogram recording rule result corruption. #7552
 * [BUGFIX] Querier: fix HTTP status code translations for remote read requests. Previously, remote-read had conflicting behaviours: when returning samples all internal errors were translated to HTTP 400; when returning chunks all internal errors were translated to HTTP 500. #7487
 * [BUGFIX] Query-frontend: Fix memory leak on every request. #7654
+* [BUGFIX] Alertmanager: Fix help message for utf-8-strict-mode. #8572
```

Comment:
Not sure if I should add this?

Reply:
So, technically the help message won't change after this - that would require a new mimir release to update the binaries / images. This will only update the website docs automatically. Is that OK?

Reply:
That's fine 👍